src/lib/Scenes/Fair/__tests__/index-tests.tsx

```diff
@@ -0,0 +1,9 @@
+import { getTextTree } from "lib/utils/getTestWrapper"
+import React from "react"
+import Fair from "../"
+
+it("Renders a fair", () => {
+  const fair = { name: "My Fair Lady Fair" }
+  const textRepresentation = getTextTree(<Fair fair={fair} />)
+  expect(textRepresentation).toContain(fair.name)
+})
```

Comment:
Adds a simple test that converts the React tree into text and verifies that the fairs name is in it

Reply:
👍